clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -146,6 +146,10 @@ export class HttpConnection implements IConnection {
                         return;
                     }
 
+                    if ((negotiateResponse as any).ProtocolVersion) {
+                        throw Error("Detected a connection attempt to an ASP.NET SignalR Server. This client only supports connecting to an ASP.NET *Core* SignalR Server.");
```

Comment:
Remove the `*`, this isn't markdown :). Add the `aka.ms/signalr-core-differences` link as well

Reply:
Also I'm updating the doc to provide some more helpful content here: https://github.com/aspnet/Docs/pull/8148